src/os_notif.eliom

```diff
@@ -146,6 +146,21 @@ VVV See if it is still needed
     let uc = Eliom_reference.Volatile.get userchannel in
     I.remove uc id
 
+  let unlisten_user ~userid (id : A.key) =
+    let state =
+      Eliom_state.Ext.volatile_data_group_state
+        ~scope:Eliom_common.default_group_scope
+        (Int64.to_string userid)
+    in
+    Eliom_state.Ext.iter_volatile_sub_states
+      ~state
+      (fun state ->
+        Eliom_state.Ext.iter_volatile_sub_states
```

Comment:
This is a way to iter on all processes of all sessions of a group.
There is an explanation in the mini-howto (but hidden at the very end of tutorials):
http://ocsigen.org/tuto/5.0/manual/how-to-iterate-on-all-sessions-for-one-user-or-all-tabs
I added some comments in the code.